cisstats.py

```diff
@@ -78,12 +82,32 @@ def from_element(self, el):
         self.oval = False if oval is None else True
         return self
 
+    def _get_rule_path(self):
+        cmd = ["find", self.repo_path, "-name", self.name, "-type", "d"]
+        process = subprocess.run(cmd, stdout=subprocess.PIPE, universal_newlines=True)
+        if process.returncode != 0:
+            print("WARNING: Rule path not found for rule: %s" % self.name)
+            return None
+        return process.stdout.strip("\n")
```

Comment:
Without shelling out to find:
```
# get the rule path (dir that contains rule.yaml) relative to the repo
rule_list = [ dp for dp,dl,fn in os.walk('applications/openshift') if dp.endswith("/" + rule_name) and len(fn) == 1 and fn[0] == 'rule.yml']
 if len(rule_list) != 1:
             return None
return rule_list[0]
```
otherwise it's the same I had as a WIP locally more or less :-)